actionmailer/lib/action_mailer/form_builder.rb

```diff
@@ -23,7 +23,7 @@ module ClassMethods
       # in the views rendered by this mailer and its subclasses.
       #
       # ==== Parameters
-      # * <tt>builder</tt> - Default form builder, an instance of ActionView::Helpers::FormBuilder
+      # * <tt>builder</tt> - Default form builder. Must be a subclass of ActionView::Helpers::FormBuilder
```

Comment:
I don't think it needs to be a subclass, it just needs to have the same interface.
To keep inline with the documentation on line 7, maybe:
```suggestion
      # * <tt>builder</tt> - Default form builder. Accepts a subclass of ActionView::Helpers::FormBuilder
```

Reply:
Yes, that's better. Thanks